### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by catap** [beetbox/beets#4144] [Ideas]
Hey,

I feel that python-3.10 support is a good reason to make a new release.

What do you think?

**Comment by sampsyo** [comment]
Yep, it's about time. We recently decided to go on a 3-month release cadence, which would put the next release on about Friday.

**Reply by catap** [reply]
Maybe it is a good idea to describe this policy somewhere?

**Reply by sampsyo** [reply]
Maybe so! We haven't done anything more than come up with the idea that this would be good to do, but any help crafting the new policy would be welcome.

**Comment by catap** [comment]
@sampsyo lets make a new release? ;)
